src/App.spec.ts

```diff
@@ -637,11 +637,12 @@ describe('App', () => {
             await next();
           });
           app.action('block_action_id', async ({ }) => { await actionFn(); });
-          app.action({ callback_id: 'message_action_callback_id' }, async ({ }) => { await actionFn(); });
-          app.action(
-            { type: 'message_action', callback_id: 'another_message_action_callback_id' },
-            async ({ }) => { await actionFn(); });
-          app.action({ type: 'message_action', callback_id: 'does_not_exist' }, async ({ }) => { await actionFn(); });
+          // app.action({ callback_id: 'message_action_callback_id' }, async ({ }) => { await actionFn(); });
```

Comment:
should probably transition these three into `app.shortcut()` method calls. this likely requires adding a `shortcutFn` fake like the others in the Arrange section.

Reply:
and we probably want to add new cases for global shortcuts as well.